libs/util/shared/src/lib/models/elasticsearch.model.ts

```diff
@@ -35,3 +35,18 @@ export type EsSearchResponse = any
 
 export type EsTemplateValues = 'y' | 'n' | 's' | 't'
 export type EsTemplateType = EsTemplateValues | EsTemplateValues[]
+
+export const ES_RESOURCES_VALUES = [
```

Comment:
Does the model serve primarily for documentation purposes, so far? I don't see it being used yet. Could an `enum` for the values make sense, to prevent hard coding the strings in the code ?

Reply:
actually i used them but with the refactoring I don't use the type.
but I hope one day we can have proper typing for ES models so I would still keep them.
unions are preferred than enum